File: app/src/main/java/cailu/org/mysurfaceview/MainActivity.java
package cailu.org.mysurfaceview;

import android.app.Activity;
import android.os.Bundle;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
//        requestWindowFeature(Window.FEATURE_NO_TITLE);
        super.onCreate(savedInstanceState);
//        setContentView(new FreeFall(this));

//        // 蜘蛛网图
//        RadarView radarView=new RadarView(this);
//        setContentView(radarView);

        //微信雷达搜索
        setContentView(R.layout.activity_main2);
    }
}

File: app/src/main/java/per/wsj/app/MainActivity.kt
package per.wsj.app

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import per.wsj.commonlib.utils.DisplayUtil
import per.wsj.commonlib.utils.LogUtil

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        LogUtil.e("width: " + DisplayUtil.getScreenWidth())
    }
}